src/lib/issue.ts

```diff
@@ -79,7 +79,7 @@ export class Issue {
     description: string,
     assignees: string[],
     tags: string[],
-    host: Host,
+    host: BaseUrl,
```

Comment:
I would suggest aligning variable name and type name to have just one term for this thing.

Reply:
216607835f223c450f8ebb6d736c9cbb377aea23
d7bcbed24e49afdc82c634f9bdf10adc99eb5545